absl/base/config.h

```diff
@@ -254,7 +254,7 @@
 // POSIX.1-2001.
 #ifdef ABSL_HAVE_MMAP
 #error ABSL_HAVE_MMAP cannot be directly set
-#elif defined(__linux__) || defined(__APPLE__) || defined(__ros__) || \
+#elif defined(__linux__) || defined(__APPLE__) || defined(__FreeBSD__) || defined(__ros__) || \
```

Comment:
https://github.com/abseil/abseil-cpp/blob/master/CONTRIBUTING.md#coding-style

We have a style-guide that code needs to adhere to. One of the rules we have is that C++ source files have an 80 char limit per line.  Can you make sure you code adheres to this?

You might be able to fix this automatically by running "git clang-format".

Reply:
Fixed